kube-controllers/Makefile

```diff
@@ -31,6 +31,10 @@ else
 CGO_ENABLED=0
 endif
 
+LDFLAGS=-ldflags "\
```

Comment:
All these shorter ones don't need to be split across multiple lines - let's just keep it on one for simplicity.

Reply:
That seems reasonable. I have addressed it.